app/src/main/java/com/helio/app/ui/blind/SingleBlindSettingsPreferencesFragment.java

```diff
@@ -38,8 +44,15 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         //Name
         assert namePreference != null;
         namePreference.setOnBindEditTextListener(editText -> editText.setInputType(InputType.TYPE_CLASS_TEXT));
+        namePreference.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
+            @Override
+            public boolean onPreferenceChange(Preference preference, Object newValue) {
+                blindName = String.valueOf(namePreference.getSummary());
+                return true;
+            }
+        });
+
```

Comment:
```suggestion
```
The way this is being done is very impractical, just remove the bit about displaying the controlled blinds for now and we can implement it properly later with ViewModels. Blindname is static and it is very weird. Please read up on app architecture before trying anything like this https://developer.android.com/jetpack/guide.